assets/templates/metaboxes/metabox.repo.git.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ +--------------------------------------------------------------------+
+ | Copyright CiviCRM LLC. All rights reserved.                        |
+ |                                                                    |
+ | This work is published under the GNU AGPLv3 license with some      |
+ | permitted exceptions and without any warranty. For full license    |
+ | and copyright information, see https://civicrm.org/licensing       |
+ +--------------------------------------------------------------------+
+ */
+
+/**
+ *
+ * @package CRM
+ * @copyright CiviCRM LLC https://civicrm.org/licensing
+ *
+ */
+
+?><!-- assets/templates/metaboxes/metabox.repo.git.php -->
+<p><?php _e('For the more adventurous, here is a list of plugins that are hosted in git repositories around the web. You may need a bit more technical confidence to install and upgrade these plugins.', 'civicrm'); ?></p>
+
+<hr/>
+
+<ul>
+  <li><em><a href="https://github.com/search?p=2&amp;q=civicrm+wordpress&amp;type=Repositories" target="_blank"><?php _e('Search GitHub for CiviCRM &amp; WordPress', 'civicrm'); ?></a></em></li>
+</ul>
+
+<hr/>
+
+<ul>
+  <li><a href="https://github.com/christianwach/civicrm-acf-integration" target="_blank">CiviCRM ACF Integration</a></li>
+  <li><a href="https://github.com/aghstrategies/civicrm-contact-directory" target="_blank">CiviCRM Contact Directory</a></li>
+  <li><a href="https://github.com/christianwach/civicrm-directory" target="_blank">CiviCRM Directory</a></li>
+  <li><a href="https://github.com/christianwach/civicrm-event-organiser" target="_blank">CiviCRM Event Organiser</a></li>
+  <li><a href="https://develop.tadpole.cc/plugins/civicrm-groups-sync" target="_blank">CiviCRM Groups Sync</a></li>
+  <li><a href="https://develop.tadpole.cc/plugins/civicrm-gutenberg-blocks" target="_blank">CiviCRM Gutenberg Blocks</a></li>
+  <li><a href="https://lab.civicrm.org/extensions/wp-civicrm-mailer" target="_blank">CiviCRM Mailer for WordPress</a></li>
+  <li><a href="https://develop.tadpole.cc/plugins/civicrm-member-sync-single-role" target="_blank">CiviCRM Member Sync Single Role</a></li>
+  <li><a href="https://develop.tadpole.cc/plugins/civicrm-network-groups" target="_blank">CiviCRM Network Groups</a></li>
+  <li><a href="https://develop.tadpole.cc/plugins/civicrm-permissions-sync" target="_blank">CiviCRM Permissions Sync</a></li>
+  <li><a href="https://github.com/mecachisenros/content-views-civicrm" target="_blank">Content Views CiviCRM</a></li>
+  <li><a href="https://github.com/agileware/content-views-civicrm-data-processor" target="_blank">Content Views CiviCRM with CiviCRM Data Processor</a></li>
+  <li><a href="https://github.com/mecachisenros/woocommerce_civicrm" target="_blank">WooCommerce CiviCRM Integration</a></li>
+  <li><a href="https://github.com/agileware/wp-civicrm-ux" target="_blank">WordPress CiviCRM UX</a></li>
+  <li><a href="https://github.com/CiviCooP/wpcivi-shared" target="_blank">WPCivi Shared</a></li>
```

Comment:
I'm not sure what the general CiviCRM approach to this is, but it feels like all external links on these metaboxes should contain `rel="noopener"`. See https://web.dev/external-anchors-use-rel-noopener/. Whilst these third-party sites are trusted, adding the attribute will protect CiviCRM site owners against potential hacks of lab.civicrm.org, develop.tadpole.cc etc

Reply:
@braders I've removed the `target="_blank"` attributes instead - they're not really necessary.

Reply:
By that I mean it's not rocket science to open links in new tabs :-)